src/vm/win32threadpool.h

```diff
@@ -1022,12 +1022,12 @@ class ThreadpoolMgr
         {
             LIMITED_METHOD_CONTRACT;
 
-	    if (CPUGroupInfo::CanEnableGCCPUGroups() && CPUGroupInfo::CanEnableThreadUseAllCpuGroups())
-            return pRecycledListPerProcessor[CPUGroupInfo::CalculateCurrentProcessorNumber()][memType];
-        else
-            // Turns out GetCurrentProcessorNumber can return a value greater than the number of processors reported by
-            // GetSystemInfo, if we're running in WOW64 on a machine with >32 processors.
-        	return pRecycledListPerProcessor[GetCurrentProcessorNumber()%NumberOfProcessors][memType];
+	        if (CPUGroupInfo::CanEnableGCCPUGroups() && CPUGroupInfo::CanEnableThreadUseAllCpuGroups())
```

Comment:
are you ok if I fix this with my next checkin? I don't want to submit another checkin just for this as it seems very difficult in general to get these checkins through due to one problem or another.